.github/workflows/nim_test.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: nim_test
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  nim_test:
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        nim-version:
+          - '1.6.12'
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: jiro4989/setup-nim-action@v1
+        with:
+          nim-version: ${{matrix.nim-version}}
+
+      - name: Build and test
+        run: |
+          nim test
+...
```

Comment:
I don't think these dots should be here.
```suggestion
```

Reply:
They are already [here](https://github.com/TheAlgorithms/Nim/blob/a2fc1bd53dcd8a320cc67074d35c02f2b73ca209/.github/workflows/check_code_format.yml#L35). I think it is good to be consistent.